src/public/app/widgets/dialogs/sort_child_notes.js

```diff
@@ -65,6 +65,17 @@ const TPL = `<div class="sort-child-notes-dialog modal mx-auto" tabindex="-1" ro
                             sort folders at the top
                         </label>
                     </div>
+                    
+                    <br />
+                    
+                    <h5>Natural Sort</h5>
+
+                    <div class="form-check">
+                        <label class="form-check-label">
+                            <input class="form-check-input" type="checkbox" name="sort-natural" value="1">
+                            sort by alphanumeric order
```

Comment:
I think this needs some more explanation. The default sorting is also technically "alphanumeric". Something like "language rule respecting sorting". Not sure how to phrase it.

Reply:
I changed it into "sort with respect to different character sorting and collation rules in different languages or regions."